modules/ossm-automatic-sidecar-injection.adoc

```diff
@@ -7,63 +7,76 @@
 [id="ossm-automatic-sidecar-injection_{context}"]
 = Enabling automatic sidecar injection
 
-When deploying an application, you must opt-in to injection by setting the `sidecar.istio.io/inject` annotation to `"true"`. Opting in ensures that the sidecar injection does not interfere with other {product-title} features such as builder pods used by numerous frameworks within the {product-title} ecosystem.
+When deploying an application, you must opt-in to injection by configuring the annotation `sidecar.istio.io/inject` in `spec.template.metadata.annotations` to `true` in the `deployment` object. Opting in ensures that the sidecar injection does not interfere with other {product-title} features such as builder pods used by numerous frameworks within the {product-title} ecosystem.
 
 .Prerequisites
 
-* Identify the deployments for which you want to enable automatic sidecar injection.
+* Identify the namespaces that are part of your service mesh and the deployments that need automatic sidecar injection.
 
 .Procedure
 
-. Open the application's deployment configuration YAML file in an editor. To find a deployment use the `oc get` command. For example, for an app called `sleep` in the `sleep` namespace, use the following command to see the resource in YAML format.
+. To find your deployments use the `oc get` command.
 +
 [source,terminal]
 ----
-$ oc get deployment sleep -o yaml
+$ oc get deployment -n <namespace>
 ----
++
+For example, to view the deployment file for the 'ratings-v1' microservice in the `bookinfo` namespace, use the following command to see the resource in YAML format.
++
+[source,terminal]
+----
+oc get deployment -n bookinfo ratings-v1 -o yaml
+----
++
+. Open the application's deployment configuration YAML file in an editor.
 
-. Add `sidecar.istio.io/inject` to the configuration YAML with a value of `"true"` in the `spec.template.metadata.annotations.sidecar.istio/inject` field. See the following example for an app called `sleep`.
+. Add `spec.template.metadata.annotations.sidecar.istio/inject` to your Deployment YAML and set `sidecar.istio.io/inject` to `true` as shown in the following example.
```

Comment:
In this case, since I'm talking about editing an actual Deployment file, I believe that the capitalization is correct (it's lowercase for general references).

See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting

Reply:
But......Is the name of the file Deployment? If not, it's a general use of the word. Like: "Create a deployment file named `ShipIt.abc`.

Reply:
OK, if this is an API object, I guess it is capitalized. :-)